packages/flutter_tools/lib/src/reporting/custom_dimensions.dart

```diff
@@ -127,6 +128,7 @@ class CustomDimensions {
   final int? hotEventReloadVMTimeInMs;  // cd 55
   final bool? commandRunEnableImpeller;  // cd 56
   final String? commandRunIOSInterfaceType; // cd 57
+  final bool? commandRunEnableEmbedderApi; // cd 58
```

Comment:
Do we really need to track this as usage data? I can't imagine more than one or two non-contributor customers ever running with this (as opposed to impeller).

Reply:
I will remove it.